crates/uv-installer/src/plan.rs

```diff
@@ -429,7 +397,11 @@ enum Specifier<'a> {
 pub struct Plan {
     /// The distributions that are not already installed in the current environment, but are
     /// available in the local cache.
-    pub local: Vec<CachedDist>,
+    pub cached: Vec<CachedDist>,
+
+    /// Any distributions that are already installed in the current environment, and can be used
+    /// to satisfy the requirements.
+    pub local: Vec<InstalledDist>,
```

Comment:
Perhaps we should rename this to `installed`?

Reply:
I think I had that initially and changed to `local` to opposed `remote`. I don't really like `remote` either since they could be on the user's machine just not in the environment... hm.

Reply:
This was for a bad reason, it collided with an `installed` variable in `Planner::build`. Resolved now.